colossalai/shardformer/policies/bert.py

```diff
@@ -237,7 +265,13 @@ def get_held_layers(self) -> List[Module]:
         return held_layers
 
     def get_shared_params(self) -> List[Dict[int, Tensor]]:
-        '''No shared params in bertmodel'''
+        model = self.model
+        if id(model.bert.embeddings.word_embeddings.weight) == id(model.cls.predictions.decoder.weight):
+            #tie weights
+            return [{
+                0: model.bert.embeddings.word_embeddings.weight,
+                self.pipeline_stage_manager.num_stages - 1: model.cls.predictions.decoder.weight
+            }]
         return []
 
     def postprocess(self):
```

Comment:
Should check whether pipeline parallel is used.

Reply:
done